changelogs/current.yaml

```diff
@@ -113,6 +113,9 @@ bug_fixes:
   change: |
     Fixed a bug that the subset load balancer will always be used even if the subset load balancer config does not
     contain any subset selector.
+- area: docker
+  change: |
+    Updated base image to `ubuntu:22.04` to fix memory issue (https://github.com/envoyproxy/envoy/issues/31248).
```

Comment:
```suggestion
    Updated base image to ``ubuntu:22.04`` to fix Redis memory issue (https://github.com/envoyproxy/envoy/issues/31248).
```

its rst so requires double backticks for literals

Reply:
not sure if the `Redis` is helpful - trying to understand exactly what the issue is

Reply:
... i guess because im thinking that if we are hitting it with this container/version combo, then others may hit the same issue eg just using ubuntu or similar

Reply:
> not sure if the `Redis` is helpful - trying to understand exactly what the issue is

Just checked with default config - same behaviour, so its not related to redis:
<img width="1897" alt="image" src="https://github.com/envoyproxy/envoy/assets/1713091/1f4e5242-7762-4efd-b6d7-15a141958514">

> ... i guess because im thinking that if we are hitting it with this container/version combo, then others may hit the same issue eg just using ubuntu or similar

I also tried to reproduce in local docker environment with no luck. But it's fully reproducible in Kubernetes clusters, I have access to.
All clusters are pretty same, specs:
```
kubernetes: vanilla v1.26.4
os: ubuntu 20.04.5
kernel: 5.15
runtime: containerd 1.6.8
```
So it could be combo of any component versions.